app/views/goodsRecord/SingleRecordView.scala.html

```diff
@@ -49,12 +51,39 @@ <h3 class="govuk-notification-banner__heading">
 
 
     <h1 class="govuk-heading-l">@messages("singleRecord.h1")</h1>
+    <h2 class="govuk-heading-m">@messages("singleRecord.h2")</h2>
 
+    <p class="govuk-body">@messages("singleRecord.status")
+
+        <td class="govuk-table__cell">
+            <strong class="govuk-tag govuk-tag--red">
+                @messages(declarableStatus.messageKey)
+            </strong>
+        </td>
+    </p>
     @if(recordLocked){
         <p class="govuk-inset-text">@messages("singleRecord.recordLocked")</p>
-    } else {
-        <p class="govuk-body">@messages("singleRecord.p1")</p>
-        <p class="govuk-body">@messages("singleRecord.p2")</p>
```

Comment:
Should we remove these from messages file?

Reply:
yes, I just removed it from the files